projects/ad463x_fmcz/src/parameters.h

```diff
@@ -47,8 +47,8 @@
 /******************************************************************************/
 /********************** Macros and Constants Definitions **********************/
 /******************************************************************************/
-#define AD463x_DMA_BASEADDR             XPAR_AXI_AD463X_DMA_BASEADDR
-#define AD463x_SPI_ENGINE_BASEADDR      XPAR_SPI_AD463X_SPI_AD463X_AXI_REGMAP_BASEADDR
+#define AD463x_DMA_BASEADDR             XPAR_AXI_AD4X3X_DMA_BASEADDR
```

Comment:
i think this change breaks the Xilinx build.

Reply:
how to solve this? since this change is needed to work with the latest HDL build

Reply:
I will re-run the build, maybe the new HDL artifacts were not yet available in the build system

Reply:
I talked to the HDL guys and they said to remove this commit and keep as it is on main, so I will do that.